src/fhirtypes/ElementDefinition.ts

```diff
@@ -1635,8 +1636,19 @@ export class ElementDefinition {
       'string',
       'uri'
     );
-    if (isEmpty(validTypes)) {
-      throw new CodedTypeNotFoundError(this.type ? this.type.map(t => t.code) : []);
+    const isBindable = this.type?.some(
+      type => fisher?.fishForMetadata(type.code, Type.Logical)?.canBind
+    );
+    if (isEmpty(validTypes) && !isBindable) {
```

Comment:
I'd suggest that `isBindable` should look at the `validTypes` before doing the potentially expensive fishing expedition. In 99.95% of the cases, the requirement for a bindable type will already be fulfilled before we even look at logical models. It also makes `isBindable` complete (i.e., it represents all possibility of the element being bindable).
```suggestion
    const isBindable = validTypes.length || this.type?.some(
      type => fisher?.fishForMetadata(type.code, Type.Logical)?.canBind
    );
    if (!isBindable) {
```